src/common/helpers.js

```diff
@@ -84,7 +84,12 @@ export function formatSentenceCase(str) {
 
 // Helper methods to ease working with ajax functions
 export const Ajax = {
-  get: (url, params = false) => {
+  get: (url, params = false, stringParams = []) => {
+    if (stringParams.length > 0) {
+      return asyncFetch(
+        `${url}?${getQueryString(params)}&${stringParams.join('&')}`
```

Comment:
what about passing the values in `stringParams` as key/values of `params` ??

Reply:
Because we are allowing more than one filter of the same category to be applied at the same time, `params` might get instantiated something like this: `'organism__name': 'HOMO_SAPIENS', 'organism__name': 'GALLUS_GALLUS'`, but keys in an object in javascript must be unique so the last filter to be applied overwrites the other filters. The stringParams is a workaround to allow multiple filters with the same key to be added to the URL

Reply:
I see, what if we modify the `getQueryString` function to process array values in that way?? So that we can do:

```
Ajax.get('/url', {'organism_name': ['HOMO_SAPIENS', 'GALLUS_GALLUS']}
```

It feels kind of strange that `get` has two parameters to receive the url arguments.